resume/src/components/Awards.js

```diff
@@ -0,0 +1,16 @@
+import React, { Component } from 'react';
+import '../App.css';
+
+function Awards(props){
+    let awards = props.data
```

Comment:
```const``` should be used for variables that will never change -- this applies for your other components as well